xbmc/addons/PVRClient.h

```diff
@@ -219,6 +219,13 @@ namespace PVR
      */
     PVR_ERROR RenameChannel(const CPVRChannelPtr &channel);
 
+    /*
+     * @brief Check if an epg tag can be recorded
+     * @param tag The epg tag
+     * @return true if the tag can be recorded
```

Comment:
doxy for param isrecordable is missing, please add

return doxy does not match actual return type. Please fix.